src/sparc/client/services/osparc.py

```diff
@@ -0,0 +1,64 @@
+import abc
```

Comment:
Not sure if abc is needed to be imported here.

Reply:
@athril thx for the input but this PR is still a **draft** and therefore **still not ready for review**! Sorry if I added you as review :-) . I normally prefix the title as well with  *WIP*.